deployments/dockerfiles/gosec/Dockerfile

```diff
@@ -1,11 +1,15 @@
-# Dockerfile used to create "husyci/gosec" image
```

Comment:
Hey, @renatoaquino! Thanks a lot for this contribution! Could you please not removing this header in the beginning of the file? 🙃

Reply:
Done!!!